### ISSUE TITLE
Error while ending call from receiver device.

### ISSUE THREAD

**Issue by Ahtsham0715** [flutter-webrtc/dart-sip-ua#484]
type '(EventStateChanged) => Null' is not a subtype of type '(EventType) => void' of 'listener'. Why this error coming while ending call from receiver side.

**Comment by Tin-Primas** [comment]
It's look like you didn't include the listener to your SipUaHelperListener. Did you include all the listeners like onNewMessage, onNewNotify, onNewReinvite, registrationStateChanged, transportStateChanged, callStateChanged

**Comment by Ahtsham0715** [comment]
yes I have all listeners. it was working few days ago but now giving this error. Do you think this is due to backend issue?.because accept call and other things are working
only hangup not working.

**Comment by Tin-Primas** [comment]
Maybe cause i'm using the library and it works fine to me. You can try and try the example app in example folder. I tested and the hangup work normally

**Comment by Ahtsham0715** [comment]
okay Thanks

**Comment by Yukari-Tryhard** [comment]
Remember to mark this issue as completed @Ahtsham0715  because the on going issues in this repo is too many right now. It's hard to follow now
